client/me/purchases/remove-purchase/index.jsx

```diff
@@ -16,6 +19,19 @@ import { isDomainRegistration, isPlan, isGoogleApps } from 'lib/products-values'
 import notices from 'notices';
 import purchasePaths from '../paths';
 import { removePurchase } from 'lib/upgrades/actions';
+import FormSectionHeading from 'components/forms/form-section-heading';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLegend from 'components/forms/form-legend';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextarea from 'components/forms/form-textarea';
+
+/**
+ * Module dependencies
+ */
+import debugFactory from 'debug';
+const debug = debugFactory( 'calypso:purchases:survey' );
```

Comment:
You aren't using debug. Might be helpful to do so at different points. But if you don't plan to use it, no need to declare it or import it.